Add a note about integrating Kong Manager with MFA (#5287)
Kong/docs.konghq.com: app/_src/gateway/kong-manager/auth/index.md

is required when RBAC is enabled. It sends HTTP cookies to authenticate
client requests and maintain session information.

> Note: Kong Manager does not directly offer 2FA, MFA, OTP or captcha/Recaptcha. However, if you configured Kong manager to use OIDC auth https://docs.konghq.com/gateway/latest/kong-manager/auth/oidc/configure/ then you could provide MFA etc via the OIDC provider.

Reviewer: ```suggestion
{:.note}
> **Note:** Kong Manager does not directly offer 2FA, MFA, OTP, or captcha/Recaptcha. 
However, if you configure Kong Manager to use [OIDC authentication](/gateway/{{page.kong_version}}/kong-manager/auth/oidc/configure/), then you could provide secondary authentication via your OIDC provider.
```
